src/gwt/src/org/rstudio/core/client/widget/DynamicIFrame.java

```diff
@@ -100,4 +101,5 @@ public final DocumentEx getDocument()
    protected abstract void onFrameLoaded();
 
    private String url_;
+   private int retryInterval_ = 50;
```

Comment:
Should we start this at 0 and/or postincrement so that the very first retry happens after 50ms (as it did originally) instead of 100ms?